packages/components/src/internal/components/forms/input/AssayTaskInput.tsx

```diff
@@ -42,16 +42,31 @@ async function loadInputOptions(assayId: number): Promise<InputOption[]> {
 }
 
 interface WorkflowTaskInputProps {
```

Comment:
can / should this extend DisableableInputProps instead of adding props to this interface?

Reply:
Good catch. Yes that works